devtools/Dockerfile

```diff
@@ -0,0 +1,159 @@
+FROM --platform=linux/amd64 mambaorg/micromamba:1.3.1-jammy
+
+
+#Docker is built on layers - Each Run is a Layer
+
+# Install dependencies
+
+USER root
+  # (otherwise python will not be found)
+ARG MAMBA_ROOT_PREFIX=/opt/conda
+ENV PYTHONDONTWRITEBYTECODE=true
+ENV BASE=$MAMBA_ROOT_PREFIX
+
+RUN apt-get update && apt-get install -y \
+    git \
+    gcc \
+    g++ \
+    make \
+    libgomp1\
+    libxrender1 \
+    libtinfo6 \
+    # Clean up
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
What does this do

Reply:
So because the original image is very slim, we need to install some packages to allow for our software to compile and run. After the installation I remove all the downloaded files as they are now installed and don't need the installers. Additionally, the removal of the apt lists is essentially the removal of the packet manager for Linux because we don't need to install anymore packages and the packet manager takes up about 90mbs if I recall correctly